weave

```diff
@@ -889,6 +918,14 @@ case "$COMMAND" in
         wait_for_status_ip $PROXY_CONTAINER_NAME 127.0.0.1 $PROXY_PORT
         echo $PROXY_CONTAINER
         ;;
+    proxy-env)
+        proxy_addr "export DOCKER_HOST="
+        exit $?
+        ;;
+    proxy-config)
+        proxy_addr "-H="
+        exit $?
+        ;;
```

Comment:
why the exit? we don't do that elsewhere. And why not make `proxy_addr` echo the address to stdout? See `container_id` for an example of a function that works that way.